charts/velero/templates/NOTES.txt

```diff
@@ -11,3 +11,55 @@ Once velero server is up and running you need the client before you can use it
 2. tar -xvf velero-{{ .Values.image.tag }}-darwin-amd64.tar.gz -C velero-client
 
 More info on the official site: https://velero.io/docs
+
+{{- /*
+  Breaking changes.
+*/}}
+
+{{- $breaking := "" }}
+{{- $breaking_title := "\n" }}
+{{- $breaking_title = print $breaking_title "\n#################################################################################" }}
+{{- $breaking_title = print $breaking_title "\n######   BREAKING: The config values passed contained no longer accepted    #####" }}
+{{- $breaking_title = print $breaking_title "\n######             options. See the messages below for more details.        #####" }}
+{{- $breaking_title = print $breaking_title "\n######                                                                      #####" }}
+{{- $breaking_title = print $breaking_title "\n######             To verify your updated config is accepted, you can use   #####" }}
+{{- $breaking_title = print $breaking_title "\n######             the `helm template` command.                             #####" }}
+{{- $breaking_title = print $breaking_title "\n#################################################################################" }}
+
+{{- if hasKey .Values "resticTimeout" }}
+{{- $breaking = print $breaking "\n\nREMOVED: resticTimeout has been removed, and it is named fsBackupTimeout" }}
+{{- end }}
+
+{{- if hasKey .Values "defaultVolumesToRestic" }}
+{{- $breaking = print $breaking "\n\nREMOVED: defaultVolumesToRestic has been removed, and it is named defaultVolumesToFsBackup" }}
+{{- end }}
+
+{{- if hasKey .Values "defaultResticPruneFrequency" }}
+{{- $breaking = print $breaking "\n\nREMOVED: defaultResticPruneFrequency has been removed, and it is named defaultRepoMaintainFrequency" }}
+{{- end }}
+
+{{- if hasKey .Values "deployRestic" }}
+{{- $breaking = print $breaking "\n\nREMOVED: deployRestic has been removed, and it is named defaultNodeAgent" }}
+{{- end }}
+
+{{- if hasKey .Values "restic" }}
+{{- $breaking = print $breaking "\n\nREMOVED: restic has been removed, and it is named nodeAgent" }}
+{{- end }}
+
+{{- if hasKey .Values.configMaps "restic-restore-action-config" }}
+{{- $breaking = print $breaking "\n\nREMOVED: restic-restore-action-config has been removed, and it is named fs-restore-action-config" }}
+{{- end }}
+
+{{- range $key, $value := .Values.configMaps }}
+{{- eq $key "fs-restore-action-config" }}
```

Comment:
Here we couldn't use `{{- if hasKey index .Values.configMaps.fs-restore-action-config.labels "velero.io/restic" }}`, or it would be parsed failed with `templates/NOTES.txt:54): bad character U+002D '-'`, so I traversed the kv.